FA22.P05.Web/ClientApp/src/pages/Listings/listingsListing.tsx

```diff
@@ -0,0 +1,53 @@
+import { useEffect, useState } from "react";
+import { ApiResponse, ListingDto } from "../../types/types";
+import axios from "axios";
+import { Button, Card, CardContent, List, Typography } from "@mui/material";
+import React from "react";
+
+export default function ActiveListings() {
+    const [listings, setListings] = useState<ListingDto[]>();
+
+   
+       useEffect(() => {
+         axios.get<ListingDto[]>("/api/listings/active").then((response) => {
+           setListings(response.data);
+           console.log(response.data);
+         });
+       }, []);
+   
```

Comment:
remove lines 17-19, only want to leave one empty line between the end of the useEffect and the const listingsToShow